### PR TITLE
Create PhoneLetterCombination.py

### LINKED ISSUE #26: Phone Letter Combinations
Before jumping into Pr first comment for assign.All Problem from Edabit. Link is mandatory to add

**Problem statement:**Given a string containing digits from 2-9 inclusive, return all possible letter combinations that the number could represent. 
**Problem Link:** https://edabit.com/challenge/npLurjMJofmFRCJwx

---
Before submitting a PR please Check some details.Check Edabit for problem task

- [x] fileName formet should be in camelcase
    example- returnTheNextNumber.py
- [x]  Add label for Problem 
- [x]  In file there should be problem statement and link to probelm
    example - 
```python
'''
    Probem Task : This program will generate list of all possible string from phone letter
    Problem Link : https://edabit.com/challenge/npLurjMJofmFRCJwx
'''
*/
def main():
    return a + b
```

This repository is vaild for HacktoberFest2020

Read before making PR - [How to contribute on Github](https://www.dataschool.io/how-to-contribute-on-github/)
Good Luck,

### PR DESCRIPTION
**Pull request by Jayshah6699**
## Description

Complete code of phone letter combination.

**Issue No.**- #26 
> Type issue number after hash(#)

## Checklist

- [x] Yes, this task assigned to me.
- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] My Filename is in camelCase format
- [x] I have added **Problem statement**, **Problem link** in file.

### CODE REVIEW

**Comment by Jayshah6699** [comment]
@aayushi-droid check

File: PhoneLetterCombination.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
+```python3
```
Comment (aayushi-droid) [inline PhoneLetterCombination.py]: backtick(`) is not used for commenting is python.
```bash
File "/home/aayushi/Documents/CPP/mark1.py", line 1
    ```python3
    ^
SyntaxError: invalid syntax
```
Use 
```python
"""python """
```

**Review by aayushi-droid — COMMENTED** [review]
Improve commenting error.
you are using backtick in line no. 1

File: PhoneLetterCombination.py (hunk 1 above)
Comment (Jayshah6699) [reply PhoneLetterCombination.py]: Done

**Review by aayushi-droid — APPROVED** [review]
now it's good

**Comment by aayushi-droid** [comment]
- [x]  Task complete #26

**Pull request merged**